io.ocrmypdf.ocrmypdf.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.ocrmypdf.ocrmypdf</id>
+
+  <name>OCRmyPDF</name>
+  <summary>OCRmyPDF adds an OCR text layer to scanned PDF files, allowing them to be searched</summary>
+
+  <developer id="io.ocrmypdf">
+      <name>OCRmyPDF Developers</name>
+  </developer>
+
+  <url type="homepage">https://github.com/ocrmypdf/ocrmypdf</url>
+  <url type="bugtracker">https://github.com/ocrmypdf/OCRmyPDF/issues</url>
+
+  <content_rating type="oars-1.1" />
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+
```

Comment:
Add https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#icons https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#categories-and-keywords it's console so there is no desktop file to get these from.